homeassistant/components/triggercmd/hub.py

```diff
@@ -0,0 +1,75 @@
+"""A 'hub' that connects several devices."""
+
+from __future__ import annotations
+
+import asyncio
+from collections.abc import Callable
+import logging
+
+from triggercmd import client
+
+from homeassistant.core import HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Hub:
+    """Hub for TRIGGERcmd."""
+
+    manufacturer = "TRIGGERcmd"
+
+    def __init__(self, hass: HomeAssistant, token: str) -> None:
+        """Init hub."""
+        self._token = token
+        self._hass = hass
+        self._name = "TRIGGERcmd"
```

Comment:
Not used

Reply:
I removed self._hass and self._name but I kept self._token for the token property.